src/handbook/customer/marketing/webinars.md

```diff
@@ -28,13 +28,14 @@ The following are the steps to produce a montly webinar.
         * Under Survey, you can decide if you want to do a survey at the end of the webinar. We ask if someone wants to be contacted by FlowFuse to discuss our services. These become MQLs in Hubspot.
    6. Setup HubSpot to accept webinar registrations.
         * Create a new static list called ‘Webinar Registrations [&lt;month> Edition]
-        * NOTE: You can only do the following changes once the previous month webinar has passed. You can not accept registrations for multiple webinars at the same time
-        * Edit Workflow called ‘Webinar Registration Integration with Zoom’. Two changes are required:
+        * Create a new form - clone an existing webinar registration form and rename it
+        * Edit Workflow called ‘Webinar Registration Integration with Zoom’. Three changes are required:
+         * Edit enrollment criteria to the new registration form submission. 
          * Edit ‘Add Contact to Zoom Webinar’ to update the new Webinar ID. Replace the current number with the number from the webinar you just created for this month. You find the Webinar ID on the main page of the Zoom webinar. NOTE: you need to  hyphens to replace the spaces in the ID number.
         * Edit ‘Add to static list’ to change the list to the new list for this month’s webinar.
         * Remember to Save the changes.
-   7. Test the web page for the webinar to see if a new registrant gets added to the Hubspot list.
-   8. Page is now ready to be published.
+   8. Test the web page for the webinar to see if a new registrant gets added to the Hubspot list.
```

Comment:
Markdown allows you to misnumber, so you don't have to update the list. You can use just '1.' every time